exchanges/tests/test_kraken.py

```diff
@@ -1,14 +1,47 @@
+import os
 import time
 import unittest
 
 from requests.exceptions import Timeout, HTTPError
 
-from exchanges import Kraken, Trade
+from exchanges import DATA_DIR, Kraken, OrderInformation, Trade
 
 
 class TestKraken(unittest.TestCase):
     """Tests for the exchanges.Kraken class"""
 
+    def test_order_info(self):
+        """Test the order_info method"""
+
+        kraken = Kraken()
+
+        # Get a closed order id.
+        closed_order_file = os.path.join(DATA_DIR, 'kraken', 'closed-order-id')
+        if os.path.exists(closed_order_file):
+            with open(closed_order_file) as f:
+                order_id = f.readline().rstrip()
+        else:
+            raise IOError('Testing the order_info method requires an order '
```

Comment:
I think for unittest warning == fail, but there might be a setting for that. For automated testing, maybe we can add specific test suite which does not test private queries? I am not sure testing the private queries will be possible, since it would require posting the private key.